src/app/static/gulpfile.js

```diff
@@ -39,4 +40,8 @@ gulp.task('copy-templates', function () {
         .pipe(gulp.dest('public'));
 });
 
-gulp.task('styles', gulp.series(gulp.parallel('sass', 'cssmap'), 'inject'));
+gulp.task('clean-css', function () {
+    return del('public/css/**', {force: true});
```

Comment:
clean the css folder before compiling the new css to avoid weird clashes and files being included twice! only happens in dev because in docker it's always compiled fresh.